consoles/VNC.pm

```diff
@@ -428,8 +438,12 @@ sub _server_initialization ($self) {
 
     my @encs = grep { $_->{supported} } @encodings;
 
-    # Prefer the higher-numbered encodings
-    @encs = reverse sort { $a->{num} <=> $b->{num} } @encs;
+    # Prefer the higher-numbered encodings, but not Tight
+    @encs = reverse sort {
+        $a->{name} eq 'Tight' ? -1 :
+          $b->{name} eq 'Tight' ? 1 :
+          $a->{num} <=> $b->{num}
+    } @encs;
```

Comment:
Maybe we should just specify them in the order we want instead of doing the sorting at runtime.